README.md

```diff
@@ -1,3 +1,94 @@
-# geomancer
+![Geomancer logo](https://i.imgur.com/Zj5Gcd6.jpg)
 
-Automated OSM Feature Engineering built on top of BigQuery
+---
+
+Geomancer is a geospatial feature engineering Python package for OpenStreetMap (OSM) data that are saved in BigQuery. This project is currently in pre-alpha and is under active development.
+
+
+# Dependencies
+
+- Python 3.6.0+
+- geoalchemy2
+- google-cloud-bigquery
+- loguru
+- numpy
+- pandas
+- pandas-gbq
+- pytz
+- sqlalchemy
+
+All of the core dependencies can be found in `requirements.in`.
+
+# Setup for Active Testers and Developers
+
+## Installation
+
+1. Clone and enter the repository.
+```
+git clone https://github.com/thinkingmachines/geomancer.git
+cd geomancer
+```
+
+2. Make a virtual environment, install pip-tools, and activate the virtual environment.
+
+Mac/Linux users:
+```
+make venv
+```
+
+Windows users:
+```
+python -m venv venv
+venv\Scripts\activate
+pip install pip-tools
+```
+
+3. Install the dependencies.
+
+Mac/Linux users:
+```
+make build
+make dev
+```
+
+Windows users:
+```
+pip install -r requirements.txt
+pip install -r requirements-dev.txt
+```
+
+Now you're all set! Go out and cast spells or even make a few of your own. 🧙‍♀️🧙‍♂️✨
+
+
+## Updating the list of dependencies
+
+1. Open up `requirements.in` or `requirements-dev.in` on a text editor and add/remove the package.
+
+2. Use pip-compile in order to update `requirements.txt` or `requirements-dev.txt` based on the contents of the `.in` file.
+
+Mac/Linux users:
+```
+make requirements.txt
+make requirements-dev.txt
+```
+
+Windows users:
+```
+venv\Scripts\pip-compile -o requirements.txt --no-header --no-annotate --no-index --no-emit-trusted-host requirements.in
+venv\Scripts\pip-compile -o requirements-dev.txt --no-header --no-annotate --no-index --no-emit-trusted-host requirements-dev.in
+```
+
+# Setup for Users (not recommended)
+
+Geomancer can be installed by running `pip install geomancer`.
+
+
+# Basic Usage
+
+All of the feature engineering functions in Geomancer are called "spells". Currently, there's only 1 spell called `distance_to_nearest`. In order to use it:
```

Comment:
API is different, and it should be working already. Wanna try it out and update the usage? You can try `DistanceToNearest` on a set of points loaded into a `pandas.DataFrame`